near-sdk/src/store/lookup_set/mod.rs

```diff
@@ -80,7 +144,18 @@ where
         T: Borrow<Q>,
         Q: BorshSerialize + ToOwned<Owned = T> + Ord,
     {
-        self.map.contains_key(value)
+        let entry_cell = self.cache.get(value.to_owned());
+        match entry_cell.get() {
```

Comment:
better to use `get_or_init` here instead of setting and ignoring the error below